app/src/main/java/com/example/notes2text/usecases/FileOpenInteractor.java

```diff
@@ -42,8 +42,14 @@ public void openFile(Context context, File fileToOpen){
                 Bundle bundle = new Bundle();
                 bundle.putSerializable("file", fileToOpen);
                 docIntent.putExtras(bundle);
-                intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
Should be docIntent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK)

Reply:
Fixed the issue, thanks.